test/test_namedtensor.py

```diff
@@ -302,35 +309,35 @@ def test_renamed_globber(self):
         named_tensor = torch.empty(1, 1, 1, 1, names=('N', 'C', 'H', 'W'))
 
         self.assertEqual(scalar.renamed(None).names, [])
-        self.assertEqual(scalar.renamed('*').names, [])
+        self.assertEqual(scalar.renamed('...').names, [])
 
         # Check that it works with unnamed tensors
-        self.assertEqual(unnamed_tensor.renamed('*').names, unnamed_tensor.names)
-        self.assertEqual(unnamed_tensor.renamed('*', 'H', 'W').names,
+        self.assertEqual(unnamed_tensor.renamed('...').names, unnamed_tensor.names)
+        self.assertEqual(unnamed_tensor.renamed('...', 'H', 'W').names,
                          [None, None, 'H', 'W'])
-        self.assertEqual(unnamed_tensor.renamed('N', '*', 'W').names,
+        self.assertEqual(unnamed_tensor.renamed('N', '...', 'W').names,
                          ['N', None, None, 'W'])
-        self.assertEqual(unnamed_tensor.renamed('N', 'C', '*').names,
+        self.assertEqual(unnamed_tensor.renamed('N', 'C', '...').names,
                          ['N', 'C', None, None])
 
         # Check that it works with named tensors
-        self.assertEqual(named_tensor.renamed('*').names, named_tensor.names)
-        self.assertEqual(named_tensor.renamed('*', 'width').names,
+        self.assertEqual(named_tensor.renamed('...').names, named_tensor.names)
+        self.assertEqual(named_tensor.renamed('...', 'width').names,
```

Comment:
in python3 these all work without the quotes?

Reply:
Yes! There's an explicit python-3 test for this behavior somewhere in this file